backend-service/src/main/java/com/smartparkinglot/backendservice/controller/ParkingLotController.java

```diff
@@ -34,12 +36,14 @@
 
 
     @DeleteMapping()
+    @Operation(summary = "USE it for deletion")
```

Comment:
[This Comment](https://github.com/bitirme-projesi-beu/backend-service/pull/14/files#r433806525) includes this one too.